librz/core/cmd_descs/cmd_seek.yaml

```diff
@@ -10,7 +10,7 @@ commands:
       - name: addr
         type: RZ_CMD_ARG_TYPE_RZNUM
         optional: true
-  - name: "s:"
+  - name: s0
```

Comment:
What about more meaningful name? `spad` or something like that?

Reply:
ok!

Reply:
Updated